babel.config.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
   presets: [
-    ['cozy-app', { presetEnv: { modules: false }, transformRuntime: { helpers: true }}]
+    [
+      'cozy-app',
+      { presetEnv: { modules: false }, transformRuntime: { helpers: true } }
+    ]
   ],
   env: {
     transpilation: {
+      presets: ['cozy-app'],
```

Comment:
Here by redefining `presets: ['cozy-app']`, I think you are overriding `{ presetEnv: { modules: false }, transformRuntime: { helpers: true }`. This is not something that we want for transpilation as we do not want to touch `imports/exports` and do not want the babel helpers to be inlined.